pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Introduction'
+weight: 50
+---
+
+The AsyncAPI Specification defines a set of files (AsyncAPI documents) required to describe the API of an application. The AsyncAPI document describes the message-driven API in accordance with the AsyncAPI Specification via JSON objects that conform to the JSON standards. (YAML, being a superset of JSON, can also be used.) It also offers a standardized approach for documenting and describing asynchronous and event-driven systems, clearly defining various components like channels and messages. Additionally, users can leverage both the AsyncAPI document and tools like the AsyncAPI Generator for code and documentation generation, enhancing efficiency and consistency in their development processes.
```

Comment:
Additionally, we use the word `components` here, but we technically call those `objects`: 
![Google Chrome_17R585e6@2x](https://github.com/asyncapi/website/assets/1083296/7378824e-21b5-4c9e-9b00-a2dd5decd5ea)
See https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.17#schema

However, depending on the context we are, we might want to call them components. I'm not sure, being completely honest 🤷

Reply:
I see, thank you for this detail... I think it makes sense to call them objects, so I have updated that line accordingly. 😄✌🏽